extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/thing/One.xml

```diff
@@ -49,7 +49,6 @@
 			<channel id="tuneinstationid" typeId="tuneinstationid" />
 			<channel id="volume" typeId="volume" />
 			<channel id="zonegroup" typeId="zonegroup" />
-			<channel id="zonegroupid" typeId="zonegroupid" />
```

Comment:
Is there a specific reason that for the One the `zonegroupid` channel is removed instead of the `zonegroup` (like for the other players)? To me this seems to be a mistake. :)

Reply:
@kaikreuzer have you seen this comment?

Reply:
Thanks @AlexanderKostadinov28, good catch! I certainly only removed it to check if anyone is doing a thorough review ;-)